docs/AQL/master03-syntax.adoc

```diff
@@ -12,8 +18,10 @@ NOTE: If no `ORDER BY` clause is specified, then the query result doesn't have a
 
 Keywords in AQL are not case sensitive, so `SELECT`, `Select`, `select`, `SeLeCt`, etc. are processed in the same way. AQL has the following reserved keywords and characters:
 
-* `SELECT`, `FROM`, `WHERE`, `ORDER BY`
```

Comment:
Possibly you could make a separate para (e.g. using 'NOTE:' or 'WARNING:) to indicate that TOP is deprecated, rather than leaving it in the main list. This is how I have shown deprecations in ADL2. Then the 'current' valid grammar listed only contains the modern version.

Reply:
done